google-cloud-translate/lib/google/cloud/translate/service.rb

```diff
@@ -110,10 +110,10 @@ def http
         # The default HTTP headers to be sent on all API calls.
         def default_http_headers
           @default_http_headers ||= {
-            "User-Agent" => "gcloud-ruby/#{Google::Cloud::Translate::VERSION}",
-            "google-cloud-resource-prefix" => "projects/#{@project}",
-            "Content-Type" => "application/json",
-            "x-goog-api-client" => "gl-ruby/#{RUBY_VERSION} " \
+            "User-Agent":                   "gcloud-ruby/#{Google::Cloud::Translate::VERSION}",
```

Comment:
This changes the keys from String to Symbol. Is that the intention?

Reply:
reverted back to spaceships

Reply:
`s/spaceships/hashrockets`